zinchenko_ihor/task_04/script_03.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+log_file="log.log"
+
+# Очищення файлу логу, якщо він існує
+> $log_file
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)